### DISCUSSION TITLE
Problems with Subfigures when rendering PDFs

### DISCUSSION THREAD

**Discussion by friosavila** [quarto-dev/quarto-cli#9683] [Q&A] [answered]
### Description

Please consider the following code:

```
---
title: "Figure Problem"
format: pdf
---

Lets make  multi figure pdf file.

:::{#fig-fig1 fig-pos="H" layout-ncol=2}

![Part1](https://i.imgflip.com/8q5rb8.jpg)

![Part2](https://i.imgflip.com/8q5rb8.jpg)

![Part3](https://i.imgflip.com/8q5rb8.jpg)
:::
```

WHen I render this document into PDF i would expect each subfigure title to be:
(a) Part1, (b) Part2, and (c) Part3
Instead I get
(a) Part1, (a) Part2, and (a) Part3

Thank you

**Comment by cscheid** [comment, marked as answer]
This is a bug (specifically we're doing a bad job with unexpected input where you have captions on the subfigures but no labels on them). In the meantime, you can add unique labels to the subfigures:

````
---
title: "Figure Problem"
format: pdf
keep-tex: true
---

Lets make  multi figure pdf file.

:::{#fig-fig1 fig-pos="H" layout-ncol=2}

![Part1](https://i.imgflip.com/8q5rb8.jpg){#fig-fig1-a}

![Part2](https://i.imgflip.com/8q5rb8.jpg){#fig-fig1-b}

![Part3](https://i.imgflip.com/8q5rb8.jpg){#fig-fig1-c}
:::
````

**Reply by cscheid** [reply]
Tracking here: https://github.com/quarto-dev/quarto-cli/issues/9688

**Reply by friosavila** [reply]
Thank you!
